src/SFA.DAS.EAS.Portal.UnitTests/Worker/EventHandlers/Paye/WhenAddedPayeSchemeEventReceived.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using NServiceBus;
+using NUnit.Framework;
+using SFA.DAS.EAS.Portal.Application.Commands;
+using SFA.DAS.EAS.Portal.Application.Commands.Paye;
+using SFA.DAS.EAS.Portal.Worker.EventHandlers.PayeScheme;
+using SFA.DAS.EmployerAccounts.Messages.Events;
+
+namespace SFA.DAS.EAS.Portal.UnitTests.Worker.EventHandlers.Paye
+{
+    [Parallelizable]
+    [TestFixture]
+    class WhenAddedPayeSchemeEventReceived
```

Comment:
If this PR is merged with CON-409, the heavy lifting for this test could be handled by `EventHandlerTestsFixture`.
Once the message handling context is populated in the event handler, that should be tested also.

Reply:
It could but no more CON-454s wait until one is in master and fix the second one or tech debt it up